### DISCUSSION TITLE
Error in loadData(), unknown element type SHELL4

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#253] [General]
While running my tensile_test.yaml file with modified tensile_test.g file, Error in loadData(), unknown element type SHELL4 is showing, please help to resolve this issue.

**Comment by oldninja** [comment]
I think you need to make sure you only have exported solid elements for your mesh.

**Reply by Nikhil23121998** [reply]
Thank you.
